roles/maintenance_32_apache/tasks/main.yml

```diff
@@ -0,0 +1,269 @@
+  changed_when: no
+
+# Extract whether SSL is enabled
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Extract whether SSL is enabled"
+  ansible.builtin.shell: "{{ apache_binary_path }} -M | grep ssl"
+  register: apache_mod_ssl_output
+  ignore_errors: true
+  changed_when: no
+
+# Extract whether SSL is enabled
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Extract whether SSL is enabled"
+  ansible.builtin.set_fact:
+    apache_is_ssl_active: "{{ 'true' if 'ssl_module' in apache_mod_ssl_output.stdout else 'false' }}"
+  changed_when: no
+
+# Extract all config file regexes
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Extract config path regexes from main apache config"
+  ansible.builtin.shell:
+    cmd: "grep -r '^Include' {{ apache_config_file_name }} | awk '{printf \"/etc/httpd/%s\\n\", $2}'"
+  register: apache_included_config_paths_out
+  ignore_errors: true
+  changed_when: no
+
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Extract config path regexes from main apache config"
+  ansible.builtin.set_fact:
+    apache_included_config_paths: "{{ apache_included_config_paths_out.stdout_lines }}"
+  ignore_errors: true
+  changed_when: no
+
+# Find all included error log files in all configs
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Extract log paths from apache configs"
+  ansible.builtin.shell:
+    cmd: "grep -rhE '^ErrorLog\\|^CustomLog' {{ item }} | awk '{print $2}'"
+  loop: "{{ apache_included_config_paths }}"
+  register: apache_log_files_out
+  changed_when: no
+  ignore_errors: true
+
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Extract log paths from apache configs"
+  ansible.builtin.set_fact:
+    apache_log_files: "{{ apache_log_files_out.results | map(attribute='stdout_lines') | flatten }}"
+  changed_when: no
+  ignore_errors: true
+
+# Add the default log paths to the list
+# There might also be module specific ones in this list
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Extract log paths from apache configs"
+  ansible.builtin.set_fact:
+    apache_log_files: "{{ apache_log_files | default([]) + [
+                  '/var/log/{{ apache_binary_name }}/error_log',
+                  '/var/log/{{ apache_binary_name }}/access_log',
+                  '/var/log/{{ apache_binary_name }}/ssl_engine_log',
+                  '/var/log/{{ apache_binary_name }}/rewrite.log',
+                  '/var/log/{{ apache_binary_name }}/debug.log']
+                }}"
+  changed_when: no
+  ignore_errors: true
+
+# Find all included .pem, .crt, .key, .csr files in all apache configs
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Extract cert paths from apache configs"
+  ansible.builtin.shell:
+    cmd: "grep -rhE '^(SSLCertificateFile|SSLCertificateKeyFile)' {{ item }} | awk '{print $2}'"
```

Comment:
Some older installations (<2.4.8) may still be using `SSLCertificateChainFile`, e.g. EL7.
Also, do we actually need the key files? (e.g. the keys don't expire; there's no validity to check)